micropython/bluetooth/aioble-central/manifest.py

```diff
@@ -1,4 +1,4 @@
-metadata(version="0.2.0")
+metadata(version="0.3.0")
```

Comment:
@jimmo not sure this is the correct way to bump the version for aioble.  Maybe it should be done to all components?  (And maybe 0.2.1?, it could be considered a bug fix, it doesn't add anything to the API.)

Reply:
Let's go for 0.2.1, since it doesn't change any API.

And the super package `aioble` also needs its version number bumped, due to its dependency on `aioble-central`.